### PR TITLE
implementation of the UI of the movie project

### PR DESCRIPTION
**Pull request by Juliana-Dimeglio**
Implementation of "Movies Project". I created a grid view that contains the images of the movies
image

### CODE REVIEW

File: lib/src/ui/movie_list.dart [hunk 1]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Popular Movies'),
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: missed a trailing comma here

File: lib/src/ui/movie_list.dart [hunk 2]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
...
+      ),
+      body: StreamBuilder(
+        stream: movieBloc.allMovies,
+        builder: (context, AsyncSnapshot<Movie> snapshot) {
+          if (snapshot.hasData) {
+            return buildList(snapshot.data);
+          } else if (snapshot.hasError) {
+            return Text(snapshot.error.toString());
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: missed a trailing comma here

File: lib/src/ui/movie_list.dart [hunk 3]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
...
+          }
+          return Center(child: CircularProgressIndicator());
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: missed a trailing comma here

File: lib/src/ui/movie_list.dart [hunk 4]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
...
+  Widget buildList(Movie? data) {
+    return GridView.builder(
+        itemCount: data?.results.length,
+        gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+            crossAxisCount: NumericConstants.crossAxisCount),
+        itemBuilder: (BuildContext context, int index) {
+          return GridTile(
+            child: Container(
+              margin:
+                  EdgeInsets.symmetric(vertical: NumericConstants.marginImages),
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: missed a trailing comma here

File: lib/src/ui/movie_list.dart [hunk 5]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
...
+        },
+      ),
+    );
+  }
+
+  Widget buildList(Movie? data) {
+    return GridView.builder(
+        itemCount: data?.results.length,
+        gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+            crossAxisCount: NumericConstants.crossAxisCount),
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: missed a trailing comma here

File: lib/src/ui/movie_list.dart [hunk 6]
```diff
@@ -25,6 +28,40 @@ class MovieListState extends State<MovieList> {
...
+        itemBuilder: (BuildContext context, int index) {
+          return GridTile(
+            child: Container(
+              margin:
```
Comment (MarcosCGlobant) [inline lib/src/ui/movie_list.dart]: ctrl + alt + L should fix this identation

Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

File: lib/src/ui/movie_list.dart (hunk 5 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

File: lib/src/ui/movie_list.dart (hunk 4 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

File: lib/src/ui/movie_list.dart (hunk 3 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

File: lib/src/ui/movie_list.dart (hunk 2 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

File: lib/src/ui/movie_list.dart (hunk 1 above)
Comment (Juliana-Dimeglio) [reply lib/src/ui/movie_list.dart]: fixed

**Review by eis6aer — APPROVED** [review]
LGTM

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by marianosalvetti — APPROVED** [review]

**Review by FranciscoGaraventa — APPROVED** [review]
LG

**Pull request merged**